src/unix/core.c

```diff
@@ -125,8 +125,8 @@ void uv_close(uv_handle_t* handle, uv_close_cb close_cb) {
     uv__check_close((uv_check_t*)handle);
     break;
 
-  case UV_IDLE:
-    uv__idle_close((uv_idle_t*)handle);
+  case UV_SPIN:
+    uv__spin_close((uv_spin_t *) handle);
```

Comment:
For clarity, do you mean the rename from 'uv_idle_t' or the fact that there is a space before `handle` and `*`?